### PR TITLE
Add example and discussion of containerness to readme

### PR DESCRIPTION
**Pull request by oxinabox**
I always want a basic example in the readme, so I don't have to click the docs,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,37 @@
...
 
 Note: If the amount of data you have to store is very small, relative to the metadata about your environment, then JLSO is a pretty suboptimal format.
+
+
+### Basic Example:
+
+```
+julia> using JLSO, Dates
+
+julia> JLSO.save("breakfast.jlso", "food"=>"☕️🥓🍳", "cost"=>11.95, "time"=>Time(9,0))
```
Comment (rofinn) [inline README.md]:
```suggestion
julia> JLSO.save("breakfast.jlso", "food" => "omelette", "cost" => 11.95, "time" => Time(9, 0))
```

File: README.md [hunk 2]
```diff
@@ -6,16 +6,37 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/github/invenia/JLSO.jl?svg=true)](https://ci.appveyor.com/project/invenia/JLSO-jl)
 [![Codecov](https://codecov.io/gh/invenia/JLSO.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/invenia/JLSO.jl)
 
-JLSO is a storage container for serialized Julia objects.
+JLSO is a storage container for serialized Julia objects.
```
Comment (rofinn) [inline README.md]: I'm guessing the `index.md` file should be using the README?

File: README.md (hunk 1 above)
Comment (oxinabox) [reply README.md]: Why do you hate fun?

Comment (oxinabox) [reply README.md]:
```suggestion
julia> JLSO.save("breakfast.jlso", "food" => "☕️🥓🍳", "cost" => 11.95, "time" => Time(9, 0))
```

File: README.md (hunk 2 above)
Comment (oxinabox) [reply README.md]: I have often made that argument yes.
that the docs index.md and the readme should have the same info.

I have set it up before on some repos.

File: README.md (hunk 1 above)
Comment (rofinn) [reply README.md]: Emojis and unicode characters are fine to use in specific contexts, but I don't think that's appropriate for a general usage library.

Comment (oxinabox) [reply README.md]: Not even in an example in the readme?
It isn't like I am using them as a function name.
(today)

Comment (rofinn) [reply README.md]: A readme is less bad, but it could make it harder for people to help maintain the documentation.

Comment (oxinabox) [reply README.md]: It makes me happy to see emoji in examples.
idk why but I get a small amount of joy out of it.

**Comment by oxinabox** [comment]
Can has?
Or you want change readme to boring words?
If so then so be it

**Comment by rofinn** [comment]
Alright, I've viewed this on multiple operating system and editors and they all seem to support emojis so I guess this is fine.

**Pull request merged**